src/planning/path_planner/src/parameterized_spline.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Autoware Foundation
```

Comment:
Is this Autoware code? I'm not seeing it in their repo.

Reply:
no, I initially copied their class and forgot to remove this comment, oops